Calendar/BookingHelpers/ExcelWrite.ps1

```diff
@@ -0,0 +1,123 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#========================
+#Excel needs Scriptlets helper functions to be loaded, in order to have access to graph and install Export-Excel module
+#========================
+
+. $PSScriptRoot\GraphInstaller.ps1
+. $PSScriptRoot\BookingGenericFunctions.ps1
+. $PSScriptRoot\BookingMBHelpers.ps1
+. $PSScriptRoot\BookingStaffHelpers.ps1
+. $PSScriptRoot\BookingServiceHelpers.ps1
+. $PSScriptRoot\BookingCustomQuestionHelpers.ps1
+. $PSScriptRoot\BookingStaffLogHelper.ps1
+
+function GetExcelParams($path, $tabName) {
+
+    return @{
+        Path                    = $path
+        FreezeTopRow            = $true
+        Verbose                 = $false
+        TableStyle              = "Medium3"
+        WorksheetName           = $tabName
+        TableName               = $tabName
+        FreezeTopRowFirstColumn = $false
+        AutoFilter              = $true
+        AutoNameRange           = $true
+        Append                  = $true
+        AutoSize                = $true
+        Title                   = "Bookings Support Data Collector"
+        ConditionalText         = $ConditionalFormatting
+    }
+}
+
+function ExcelWrite {
+    param($identity)
+
+    Write-Host -ForegroundColor Gray "Checking Excel, Graph and bookings modules"
+    CheckExcelModuleInstalled
+    CheckGraphAuthModuleInstalled
+    CheckGraphBookingsModuleInstalled
+
+    $modulesInstalled = CheckGraphModulesInstalled
+    Write-Verbose "Modules installed: $modulesInstalled"
+    if ($modulesInstalled -eq $false) {
+        Write-Host -ForegroundColor Red "Graph modules are not installed. Exiting..."
+        exit
+    }
+
+    Connect-MgGraph -Scopes "User.Read.All", "Bookings.Read.All" -NoWelcome
+
+    Write-Host -ForegroundColor Green "Exporting to Excel..."
+
+    #Compose File name formatted as: BookingsSummary_<IdentityName>_<yyyy-MM-dd>.xlsx
+    $IdentityName = SplitIdentityFromEmail -email $identity
+    $DateStringToFileName = (Get-Date).ToString("yyyy-MM-dd")
+    $path = "BookingsSummary_$($IdentityName)_$($DateStringToFileName).xlsx"
+    Remove-Item -Path $path -Force -EA Ignore
+
+    $ExcelParamsArray = GetExcelParams -path $path -tabName "Business"
+
+    Write-Host "Exporting Booking Business"
+    $BusinessData = GetGraphBookingBusiness -Identity $identity
```

Comment:
Here & everywhere
either these should be a sentence and end in a period, 
or it is an active item so should have the "..."

also think if you want these with the -ForgroundColor of Blue or Green as it is successfully writing output.

Reply:
Only the start has the ... to indicate active items. The ones below will have only one period to indicate action performed.